packages/pigeon/tool/shared/test_runner.dart

```diff
@@ -13,17 +13,19 @@ import 'test_suites.dart';
 
 /// Runs the given tests, printing status and exiting with failure if any of
 /// them fails.
-Future<void> runTests(List<String> testsToRun) async {
+Future<void> runTests(List<String> testsToRun, {bool skipGen = false}) async {
```

Comment:
Prefer bools that are affirmative to avoid double-negatives. So this should be `runGeneration = true`, and the condition below reversed.